### PR TITLE
fix: Remove warning when using environment destinations

### PR DESCRIPTION
**Pull request by tomfrenken**
This PR is a follow-up to our CAP meeting, where we decided to remove the warning.
It removes the warning and adapts the tests to it.

### CODE REVIEW

**Comment by tomfrenken** [comment]
I just saw that we created an issue for this [here](https://github.com/SAP/cloud-sdk-backlog/issues/513), so I guess we still have to decide which approach we chose, don't merge until its decided then.

**Comment by tomfrenken** [comment]
Since we have now decided to use this approach, it's fine to merge again.

File: packages/connectivity/src/scp-cf/destination/destination-from-env.spec.ts [hunk 1]
```diff
@@ -101,11 +101,11 @@ describe('env-destination-accessor', () => {
       mockDestinationsEnv(destinationFromEnv);
 
       const logger = createLogger('env-destination-accessor');
-      const warnSpy = jest.spyOn(logger, 'warn');
+      const warnSpy = jest.spyOn(logger, 'info');
```
Comment (florian-richter) [inline packages/connectivity/src/scp-cf/destination/destination-from-env.spec.ts:104]:
```suggestion
      const infoSpy = jest.spyOn(logger, 'info');
```

**Review by florian-richter — APPROVED** [review]
Consider my tiny suggestion, otherwise LGTM.

**Pull request merged**